sdks/python/apache_beam/coders/standard_coders_test.py

```diff
@@ -193,7 +193,9 @@ class StandardCodersTest(unittest.TestCase):
       value_parser: ShardedKey(
           key=value_parser(x['key']), shard_id=x['shardId'].encode('utf-8')),
       'beam:coder:custom_window:v1': lambda x,
-      window_parser: window_parser(x['window'])
+      window_parser: window_parser(x['window']),
```

Comment:
Hmm, why did we have to update this existing test ? (just to make sure that there's no backwards compatibility issue here).

Reply:
This is the python file that actually reads the yaml, and runs the tests. I needed to register the nullable coder in it, so it knows what the nullable urn points to